packages/adsum-carousel-asia/README.md

```diff
@@ -0,0 +1,87 @@
+# Carousel component
+
+![image](https://user-images.githubusercontent.com/6003532/39986925-bebf3d72-5795-11e8-91bb-ae8235ac5896.png)
+
+[Live examples here](https://adactivesas.github.io/adsum-react-components/packages/adsum-carousel/examples/index.html)
+
+## Getting started
+
+```javascript
+    npm i --save @adactive/arc-carousel
+```
+OR
+```javascript
+    yarn add @adactive/arc-carousel
+```
+
+```javascript
+    import AdsumCarousel from "@adactive/arc-carousel"
+     ...
+    <AdsumCarousel 
+        isOpen=true 
+        medias=[]
+        onTouchToNavigate={this.onTouchToNavigate}
+    />
+```
+
+### Props
+ 
+```javascript
+type PropsType = {|
+    +isOpen: boolean,
+    +medias: Array<MediaType>,
+    +onMediaTouch: (MediaType) => void,
+    +carouselOptions?: Object,
+    +style?: CSSStyleDeclaration
+|};
+
+static defaultProps = {
+    isOpen: false,
+    medias: [],
+    onMediaTouch: null,
+    carouselOptions: {
+        dragging: false,
+        swiping: false,
+        autoplayInterval: 10000,
+        speed: 1000,
+        renderCenterLeftControls: null,
+        renderCenterRightControls: null,
+        renderCenterBottomControls: null,
+        renderBottomCenterControls: null,
+        arrows: false,
+        pauseOnHover: false,
+        slidesToShow: 1,
+        slidesToScroll: 1,
+        adaptiveHeight: true,
+        wrapAround: true,
+        autoSlide: false,
+        autoSlideInterval: 10000,
+    }
```

Comment:
In **AdsumCarousel.js** your defaultProps are :
```
static defaultProps = {
        isOpen: false,
        medias: [],
        onMediaTouch: null,
        carouselOptions: {
            dragging: false,
            swiping: false,
            speed: 1000,
            renderCenterLeftControls: null,
            renderCenterRightControls: null,
            renderCenterBottomControls: null,
            renderBottomCenterControls: null,
            arrows: false,
            pauseOnHover: false,
            slidesToShow: 1,
            slidesToScroll: 1,
            adaptiveHeight: true,
            wrapAround: true,
        },
        dynamicAutoPlayInterval: false,
        style: {},
        ButtonModalForImage: null,
        autoSlide: false,
        autoSlideInterval: 10000,
    };
```
Shouldn't it be the same in your README file ?

Reply:
oh, I thought I put it already